GNUmakefile

```
@@ -113,6 +113,14 @@ test_smoke_simulated: ## Run integration smoke tests, only using simulated netwo
 	--keep-going --trace --randomize-all --randomize-suites \
 	--progress --focus @simulated $(args) ./integration-tests/smoke
 
+.PHONY: test_soak_ocr
+test_soak_ocr: ## Run the OCR soak test
+	cd ./integration-tests && go test -v -run ^TestOCRSoak$$ ./soak -count=1 && cd ..
```

Is it necessary to `cd` back and forth? Or would this work the same?
```suggestion
	go test -v -run ^TestOCRSoak$$ ./integration-tests/soak -count=1
```

---

go.mod gets mad 
```
main module (github.com/smartcontractkit/chainlink) does not contain package github.com/smartcontractkit/chainlink/integration-tests/soak
```

This was a quick workaround in lieu of finding anything more elegant after a couple of minutes of Googling.

---

Yeah I thought that might be the case. nbd for now, but maybe long term we should consider moving these to a separate makefile within that module. :shrug: